pkg/daemon/daemon.go

```diff
@@ -1079,3 +1088,42 @@ func tryCreateNMUdevRule() error {
 	}
 	return nil
 }
+
+func trySetOvsHWOL(newState bool) (bool, bool, error) {
+	glog.V(2).Infof("trySetOvsHWOL()")
+	desiredState := strconv.FormatBool(newState)
+	var output bytes.Buffer
+	var currentState string
+
+	exit, err := utils.Chroot("/host")
+	if err != nil {
+		glog.Errorf("trySetOvsHWOL(): failed to chroot to host: %v", err)
+		return false, false, err
+	}
+	defer exit()
+
+	// if we are already in desired state, do nothing
+	cmd := exec.Command("ovs-vsctl", "get", "Open_vSwitch", ".", "other_config:hw-offload")
+	cmd.Stdout = &output
+	if err := cmd.Run(); err != nil {
+		// if ovs has not previously been configured this record may not yet exist
+		glog.Errorf("trySetOvsHWOL(): failed to get current Open_vSwitch hw-offload mode : %v", err)
+		currentState = "false"
+	} else {
+		currentState = output.String()
+	}
```

Comment:
Would also mean we don't need to convert the bool into a string. It makes a bit more sense to compare bool with bool instead of string that should look like a bool with a string that was created from a bool.

Reply:
Added "func OvsHWOLisEnabled()"